src/index.js

```diff
@@ -41,14 +42,22 @@ function drawGraphics3d(
 	// to avoid overflow when a tick numbers is out of the parent element
 	container.style.height = canvasSize + 10 + 'px';
 
+	const extent = calculateExtent(elements);
+
 	// where the camera is looking (initialized on center of the scene)
 	const focus = new Vector3(
 		0.5 * (extent.xmin + extent.xmax),
 		0.5 * (extent.ymin + extent.ymax),
 		0.5 * (extent.zmin + extent.zmax)
 	);
 
-	const viewPoint = new Vector3(...viewpoint).sub(focus);
+	// scale the viewpoint so the camera doesn't be inside the bounding box
+	const viewPoint = new Vector3(
+		viewpoint[0] * (extent.xmax - extent.xmin),
```

Comment:
@mmatera here in Mathics Core this is the following:
```py
viewpoint * max([xmax - xmin, ymax - ymin, zmax - zmin])
```

But I think this makes more sense (as we are scaling each part of the viewpoint by its respective extent length.

Also, this method works well for me:
![image](https://user-images.githubusercontent.com/62714153/124527073-efc1da00-ddda-11eb-8d9c-54f35f39391f.png)
(the original viewpoint was `[2, -4, 4]` and the spheres are between the coordinate `(100, 100, 100)` and the `(100, 100, 100)`)

Reply:
Oh, I found my answer:  
The Mathics Core approach is better when we have spheres between `(0, 0, 0)` and the `(100, 1000, 100)`.

So let's change the code!